### PR TITLE
Create homepage overview section muya

### PR DESCRIPTION
**Pull request by melmuya**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

- [#4  ] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/ts/index.ts
Comment (paulkahura) [inline src/ts/index.ts:1]: don't create pr with console.logs in your code

File: src/css/style.css
Comment (paulkahura) [inline src/css/style.css:1]: use rem/em over px

**Review by paulkahura — CHANGES_REQUESTED** [review]
This is a nice attempt but attach the images for mobile, desktop and tablet for review. Also in the template to check a box use `[x]` instead of `[#]`

**Comment by JAPHETHNYARANGA** [comment]
Good attempt but this issue has been closed . Please pick another open issue. Hint; don't delete code already in a file when working on it.

**Pull request closed without merging**